examples/http-response-interceptor/http_response_interceptor.md

```diff
@@ -1,6 +1,6 @@
 # HTTP service - Response interceptor
 
-The `http:ResponseInterceptor` is used to intercept the response and execute some custom logic. A `ResponseInterceptor` is a service object with a remote method called `interceptResponse`, which is executed before dispatching the response to the client. A `ResponseInterceptor` can be created from a service class, which includes the `http:ResponseInterceptor` service type. Then, this service object can be engaged at the listener level or service level by using the `interceptors` field in the configurations. This field accepts an array of interceptor service objects as an interceptor pipeline, and the interceptors are executed in the order in which they are placed in the pipeline. Use `ResponseInterceptors` to execute some common logic such as logging, header manipulation, state publishing, etc., for all outbound responses.
+The `http:ResponseInterceptor` is used to intercept the response and execute some custom logic. A `ResponseInterceptor` is a service object with a remote method called `interceptResponse`, which is executed before dispatching the response to the client. A `ResponseInterceptor` can be created from a service class, which includes the `http:ResponseInterceptor` service type. Then, this service object can be engaged at the listener level by using the `interceptors` field in the `http:ListenerConfiguration` or at service level by using the `http:InterceptableService` in the service definition. These accept an interceptor service object or an array of interceptor service objects as an interceptor pipeline, and the interceptors are executed in the order in which they are placed in the pipeline. Use `ResponseInterceptors` to execute some common logic such as logging, header manipulation, state publishing, etc., for all outbound responses.
```

Comment:
```suggestion
The `http:ResponseInterceptor` is used to intercept the response and execute custom logic. A `ResponseInterceptor` is a service object with a remote method called `interceptResponse`, which is executed before dispatching the response to the client. A `ResponseInterceptor` can be created from a service class, which includes the `http:ResponseInterceptor` service type. 

This service object can be engaged at the listener level by using the `interceptors` field in the `http:ListenerConfiguration` or at the service level by using the `http:InterceptableService` in the service definition. These accept an interceptor service object or an array of interceptor service objects as an interceptor pipeline and the interceptors are executed in the order in which they are placed in the pipeline. 

Use `ResponseInterceptors` to execute common logic such as logging, header manipulation, state publishing, etc., for all outbound responses.
```

This paragraph is too big and should be broken down to multiple for improved clarity.